localization/english/wc_buildings_l_english.yml

```diff
@@ -269,3 +275,632 @@
+ building_argent_tournament_grounds_01:0 "Argent Tournament Fields"
+ building_argent_tournament_grounds_01_desc:0 "A clearing is kept open year-round to facilitate the training of riders and knights."
+ building_argent_tournament_grounds_02:0 "Argent Tournament Grounds"
+ building_argent_tournament_grounds_02_desc:0 "With jousting lists and stands for the public, these tournament grounds attract a huge amount of contestants."
+ building_argent_tournament_grounds_03:0 "Royal Argent Tournament Arenas"
+ building_argent_tournament_grounds_03_desc:0 "Upgraded with seats for royal guests and a larger fairground, the tournament arena is truly impressive."
+ 
+ building_type_nordrassil:0 "Nordrassil"
+ building_type_nordrassil_desc:0 "Meaning "Crown of the Heavens", it is the original World Tree planted on the summit of Mount Hyjal after the War of the Ancients, grown from a seed of the Mother Tree G'Hanir."
+ building_nordrassil:0 "Nordrassil"
+ building_nordrassil_desc:0 "Meaning "Crown of the Heavens", it is the original World Tree planted on the summit of Mount Hyjal after the War of the Ancients, grown from a seed of the Mother Tree G'Hanir."
+ building_type_teldrassil:0 "Teldrassil"
+ building_type_teldrassil_desc:0 "Meaning "Crown of the Earth", it was an unsuccessful attempt by Archdruid Fandral Staghelm and his compatriots to restore the night elves' immortality, lost with the near destruction of Nordrassil."
+ building_teldrassil:0 "Teldrassil"
+ building_teldrassil_desc:0 "Meaning "Crown of the Earth", it was an unsuccessful attempt by Archdruid Fandral Staghelm and his compatriots to restore the night elves' immortality, lost with the near destruction of Nordrassil."
+ building_type_andrassil:0 "Andrassil"
+ building_type_andrassil_desc:0 "Meaning "Crown of the Snow", it was grown from a branch taken from Nordrassil by Fandral Staghelm and his druids in order to stop the spread of saronite in the northern continent."
+ building_andrassil:0 "Andrassil"
+ building_andrassil_desc:0 "Meaning "Crown of the Snow", it was grown from a branch taken from Nordrassil by Fandral Staghelm and his druids in order to stop the spread of saronite in the northern continent."
+ building_type_vordrassil:0 "Vordrassil"
+ building_type_vordrassil_desc:0 "Meaning "Broken Crown", they are the remains of a World Tree located in the Grizzly Hills of southern Northrend."
+ building_vordrassil:0 "Vordrassil"
+ building_vordrassil_desc:0 "Meaning "Broken Crown", they are the remains of a World Tree located in the Grizzly Hills of southern Northrend."
+ building_type_shaladrassil:0 "Shaladrassil"
+ building_type_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
+ building_shaladrassil:0 "Shaladrassil"
+ building_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
```

Comment:
You could do the same down here as well, have one line for the description and then use it on both the _desc lines for the building and building_type. Just a thought since you do it with the mines etc as well, but can stay this way, too.